app/Http/Resources/EntityPermissionTestResource.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class EntityPermissionTestResource extends EntityChild
```

Comment:
I'm not sure why this class is needed (and why it would extend EntityChild). In the controller, you can just 
`return response()->json($results);`